lib/js-file.js

```diff
@@ -43,6 +44,29 @@ var JsFile = function(filename, source, tree, options) {
 
 JsFile.prototype = {
     /**
+     * Returns the first line break character encountered in the file.
+     * Assumes LF if the file is only one line.
+     *
+     * @returns {String}
+     */
+    getLineBreak: function() {
```

Comment:
I called it `getLineBreak` because there *should* be consistent line breaks in a file.